### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by Gustry**
Just some notes, I hope I was not wrong

### CODE REVIEW

**Comment by 3nids** [comment]
Hi Etienne,
Nice to see someone else using it :)
Regarding your comments, I think we are using in a different way. We generally create the release from Github whenever we are ready (tests ok, translations finished). Hence, we are not tagging the commits from the CLI but rather through Github interface.
If you'd like documenting the tagging from CLI approach, it would be nice to present it as an alternative.
Works for you?

**Comment by 3nids** [comment]
gentle ping @Gustry

**Comment by Gustry** [comment]
Yes, no worries. I will do that. I wanted to check with you during the code sprint

File: README.md [hunk 1]
```diff
@@ -126,12 +132,26 @@ One can easily set up a deployment using Travis.
...
-  on:
-    tags: true
+
+  - provider: script
+    script: qgis-plugin-ci release ${TRAVIS_TAG} --github-token ${GH_TOKEN} --osgeo-username ${OSGEO_USERNAME} --osgeo-password {OSGEO_PASSWORD}
+    on:
+      tags: true
+```
+
+This assumes that you have an existing GitHub release. If you want, Travis can create the release itself. In the `deploy` section, you must add another `provider` before the script `qgis-plugin-ci`:
```
Comment (3nids) [inline README.md]:
```suggestion
This assumes that you have an existing GitHub release. 
Alternatively, Travis can create the release by using the `releases`provider:
```

File: README.md [hunk 2]
```diff
@@ -126,12 +132,27 @@ One can easily set up a deployment using Travis.
...
-    tags: true
+
+  - provider: script
+    script: qgis-plugin-ci release ${TRAVIS_TAG} --github-token ${GH_TOKEN} --osgeo-username ${OSGEO_USERNAME} --osgeo-password {OSGEO_PASSWORD}
+    on:
+      tags: true
+```
+
+This assumes that you have an existing GitHub release. 
+Alternatively, Travis can create the release by adding  a `releases`provider before the `script` provider:
```
Comment (Gustry) [inline README.md]:
```suggestion
Alternatively, Travis can create the release by adding a `releases` provider before the `script` provider:
```

**Comment by 3nids** [comment]
thanks a lot!

**Comment by Gustry** [comment]
I will still improve and check this tool in the coming few days about our plugins ;-)

**Pull request merged**